app/views/api/v1/registrations/create.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+if @saved
```

Comment:
This is the perfect case where you should try using `?:` operator, like `@saved ? json.email @user.email : json.email @user.errors`